api/tests/src/api/competition_alpha/test_competition_route_get.py

```diff
@@ -35,64 +35,96 @@ def test_competition_get_200_with_api_key(client, api_auth_token, enable_factory
     assert response_competition["open_to_applicants"] == competition.open_to_applicants
 
 
-def test_competition_get_with_instructions_200(client, api_auth_token, enable_factory_create):
-    # Create a competition with instructions
+def test_competition_get_with_instructions_200(
+    client, api_auth_token, enable_factory_create, monkeypatch_session, mock_s3_bucket
+):
+    monkeypatch_session.setattr(file_util, "_s3_config", None)
+
+    # Set environment variables for S3 configuration
+    monkeypatch_session.setenv("CDN_URL", "")  # Empty string to ensure no CDN is used
+    monkeypatch_session.setenv("S3_ENDPOINT_URL", "http://localstack:4566")
```

Comment:
Don't set this to localstack - we don't use localstack in unit tests - we should never set this value to anything but null in our unit tests. In `reset_aws_env_vars` we intentionally delete this to avoid ever using localstack.

I see I missed that you had done that on your prior PR, we need to undo that.